modules/openapi-generator/src/test/java/org/openapitools/codegen/java/spring/SpringCodegenTest.java

```diff
@@ -1232,4 +1234,37 @@ public void testExtraAnnotations() throws IOException {
         TestUtils.assertExtraAnnotationFiles(outputPath + "/src/main/java/org/openapitools/model");
 
     }
+
```

Comment:
Maybe a test with Set and Map would also make sense? What do you think?

Reply:
@MelleD I agree, those cases should be tested as well. I've update PR with test for Set & Map